features/old/services/multiservice.feature

```diff
@@ -31,14 +30,49 @@ Feature: Multiservice feature
   @javascript
   Scenario: Create new product: Fail scenario error message
     Given I am logged in as provider "foo.3scale.localhost"
-    And provider "foo.3scale.localhost" has "multiple_services" switch allowed
-    And service discovery is not enabled
+      And provider "foo.3scale.localhost" has "multiple_services" switch allowed
```

Comment:
Did you indent these manually or did you use a formatter? If so, which one?

Reply:
No formatter is used, I have done it manually. it looks more readable and better and also in other specs also intended same  way.

Reply:
I see, generally it's a bad idea to format code manually: it's too much time consuming and not necessarily consistent. Also, this is not how cucumbers are supposed to be written, it makes it harder to maintain a clean code. See examples at https://cucumber.io/docs/gherkin/reference/

If you're using VS code I recommend extension `alexkrechik.cucumberautocomplete` which includes document formatting.

Reply:
I use Sublime

Reply:
What's wrong is doing manually and this file precisely shows inconsistent indentation because of that. I like to use the default code style, which is the one in their documentation https://cucumber.io/docs/gherkin/reference/.

Reply:
Most likely.

Reply:
I tried installing package but its not working for me

Reply:
I cannot  even found online gherkins formatter

Reply:
reverted and pushed